src/shims-tsx.d.ts

```diff
@@ -0,0 +1,13 @@
+import Vue, { VNode } from 'vue'
+
+declare global {
+  namespace JSX {
+    // tslint:disable no-empty-interface
+    interface Element extends VNode {}
+    // tslint:disable no-empty-interface
+    interface ElementClass extends Vue {}
+    interface IntrinsicElements {
+      [elem: string]: any;
+    }
+  }
+}
```

Comment:
Is this needed? It seems to work without this file, and I would expect the typescript compiler to be able to configure this correctly.

Reply:
It was automatically generated by the Vue CLI when I created the component. I'll remove it when I switch to JS.